packages/table/preview/features.tsx

```diff
@@ -183,7 +183,10 @@ class EditableTable extends React.Component<{}, IEditableTableState> {
             <Column key={index} renderCell={this.renderCell} renderColumnHeader={this.renderColumnHeader} />
         ));
         return (
-            <Table numRows={7} selectionModes={SelectionModes.COLUMNS_AND_CELLS} enableFocus={true}>
+            <Table numRows={7}
+                   selectionModes={SelectionModes.COLUMNS_AND_CELLS}
+                   enableFocus={true}
+                   useInteractionBar={true}>
```

Comment:
Hmm, is `prettier` fixing/complaining about this formatting? Should be. Change to:

```tsx
<Table
    numRows={7}
    selectionModes={SelectionModes.COLUMNS_AND_CELLS}
    enableFocus={true}
    useInteractionBar={true}
>
```

Reply:
Actually the build is failing due to several lint issues.